root/roofit/roofit/.rootrc

```diff
@@ -0,0 +1 @@
+RooFit.Banner: no
```

Comment:
Could you add a comment, explaining why we need this?

Reply:
Anything that had been even linked with Roofit will automatically have a Roofit banner as a part of output. Only way to suppress it is to put a .rootrc with suppression setting in a place where binary will be executed (by default in ROOT master suppression is OFF and I didn't manage to convince to remove it, since it is a part of ownership notification).